include/aos/iam/certhandler.hpp

```diff
@@ -26,6 +24,55 @@ namespace certhandler {
  */
 constexpr auto cIAMCertModulesMaxCount = AOS_CONFIG_CERTHANDLER_MODULES_MAX_COUNT;
 
+/**
+ * Max number of certificate subscriptions:
+ *
+ * 1 per client & server + cIAMCertModulesMaxCount for message proxy
+ */
+constexpr auto cIAMCertSubsMaxCount = cIAMCertModulesMaxCount * 2 + cIAMCertModulesMaxCount;
+
+/**
+ * Certificate receiver interface.
+ */
+class CertReceiverItf {
+public:
+    /**
+     * Updates certificate.
+     *
+     * @param info certificate info.
+     */
+    void UpdateCert(const CertInfo& info)
```

Comment:
It should not be part of interface. The client should have access only to required methods.

Reply:
interface updated